### PR TITLE
add testing section to readme

### PR DESCRIPTION
**Pull request by matteo-hertel**
I've added a section in the readme for testing and how to use and install Jest

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,23 @@ As an alternative to the section above, you can run the app within a docker cont
...
 
+### Testing
+The chosen test runner is [Jest](https://facebook.github.io/jest/) a Facebook project, you can learn more about Jest in series of videos on [Egghead.io](https://egghead.io/lessons/javascript-test-javascript-with-jest).
+
+Jest can be installed both locally (package.json) or globally:
+* locally it will be installed when you run `npm install`
+* globally you can install it by using `npm install -g jest-cli`
+* (BONUS) you can also use [Yarn](https://yarnpkg.com/) as npm alternative, just install Yarn with `npm install -g yarn` and then use `yarn` to install dependencies
+
+once jest is installed you're ready to start testing, to ease everything up we provided some pre-build commands in the package.json:
```
Comment (jenkoian) [inline README.md]: Very minor but can you capitalise the O on once, I.e. The start of the sentence

**Comment by matteo-hertel** [comment]
No problem! I'll fix it as soon as I can!

**Comment by matteo-hertel** [comment]
The Readme has been changed!

**Comment by jenkoian** [comment]
Thanks again @matteo-hertel !!

**Pull request merged**